types/node-fetch/tsconfig.json

```diff
@@ -19,6 +19,7 @@
     },
     "files": [
         "index.d.ts",
+        "externals.d.ts",
```

Comment:
This should be in an `include` block, as it's only an assistive thing for this library, not a thing this library is exporting.

```
"include": ["externals.d.ts"],
```

Reply:
Using files over include appears to be required: https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/536403252?utm_source=github_status&utm_medium=notification